### PR TITLE
Refactoring Lambda Pipeline

### PR DESCRIPTION
**Pull request by allinwonder**
*Issue #210 , if available:*

*Description of changes:*

Refactored Lambda pipeline into 3 standalone pipelines 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/lambda/README.md [hunk 1]
```diff
@@ -3,15 +3,31 @@ This project uses [`aws-serverless-java-container`](https://github.com/awslabs/a
...
+## Prepare Microservice Source Code
+
+The example application consists of there Microservices, they are Greeting Service, Name Service and WebApp Service. Each service is developed as an self-contained module. WebApp service is responsible for frontend service, Greeting and Name service are responsbile for generating response to partials of to the client.
+
+To start with, please use the following commands to clone these services source code into the workspace
+
+Clone Greeting Service Source Code
+
+```
+cd ../ && mkdir -p services && cd ../services/
```
Comment (arun-gupta) [inline apps/lambda/README.md]: These instructions would not be required with the updated `CodeUri`

File: apps/lambda/README.md [hunk 2]
```diff
@@ -28,39 +44,35 @@ $ npm install -g aws-sam-local
...
-Next, from the project root folder - where the `sam.yaml` file is located - start the API with the SAM Local CLI.
+Next, from the project root folder - where the SAM templates are located - start the APIs with the SAM Local CLI.
 
 ```bash
-cd aws-compute-options/apps/lambda
-sam local invoke -t sam.yaml -e test/find-all-names-event.json NamesFunction
-sam local invoke -t sam.yaml -e test/find-name-event.json NamesFunction
-sam local invoke -t sam.yaml -e test/greeting-event.json GreetingFunction
+cd app/lambda
+sam local invoke -t NameSam.yaml -e test/find-all-names-event.json NamesFunction
```
Comment (arun-gupta) [inline apps/lambda/README.md]: File names would be updated

File: apps/lambda/README.md [hunk 3]
```diff
@@ -28,39 +44,35 @@ $ npm install -g aws-sam-local
...
 
 ```bash
-$ cd aws-compute-options/apps/lambda
-$ sam local start-api --template sam.yaml --env-vars test/env-mac.json
-
-...
-Mounting com.sapessi.jersey.StreamLambdaHandler::handleRequest (java8) at http://127.0.0.1:3000/{proxy+} [OPTIONS GET HEAD POST PUT DELETE PATCH]
-...
+cd apps/lambda
+sam local start-api --template GreetingSam.yaml --port 3000
```
Comment (arun-gupta) [inline apps/lambda/README.md]: File names to be updated

File: apps/lambda/README.md [hunk 4]
```diff
@@ -28,39 +44,35 @@ $ npm install -g aws-sam-local
...
 
 ```bash
-$ cd aws-compute-options/apps/lambda
-$ sam local start-api --template sam.yaml --env-vars test/env-win.json
-
-...
-Mounting com.sapessi.jersey.StreamLambdaHandler::handleRequest (java8) at http://127.0.0.1:3000/{proxy+} [OPTIONS GET HEAD POST PUT DELETE PATCH]
-...
+cd apps/lambda
+sam local start-api --template GreetingSam.yaml --port 3000
```
Comment (arun-gupta) [inline apps/lambda/README.md]: File names to be updated

File: apps/lambda/WebAppSam.yaml [hunk 5]
```diff
@@ -0,0 +1,76 @@
+AWSTemplateFormatVersion: '2010-09-09'
```
Comment (arun-gupta) [inline apps/lambda/WebAppSam.yaml]: File names to be updated

File: apps/lambda/WebAppSam.yaml [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    Type: AWS::SSM::Parameter::Value<String>
+    Default: NameApiEndpoint
+    Description: Name Service API Gateway Endpoint
+Resources:
+  WebAppFunction:
+    Type: 'AWS::Serverless::Function'
+    Properties:
+      Handler: org.aws.samples.compute.webapp.WebappHandler::handleRequest
+      Runtime: "java8"
+      CodeUri: ../../services/WebApp/target/webapp.zip
```
Comment (arun-gupta) [inline apps/lambda/WebAppSam.yaml]: Location to be updated

**Comment by arun-gupta** [comment]
Also the port numbers are:

- `3000` for `webapp`
- `3001` for `greeting`
- `3002` for `name`

File: apps/lambda/README.md (hunk 1 above)
Comment (allinwonder) [reply apps/lambda/README.md]: moved lambda codeuri path change into pipeline

**Pull request merged**